app/views/messages/_new_message.html.erb

```diff
@@ -0,0 +1,22 @@
+<form novalidate>
```

Comment:
Sending a message is the exact same thing whether it's a new message or continuing a conversation, with the single exception of whether you can set who the message is going to, so this one partial handles both cases with a simple If/Else.